examples/dask/.saturn/saturn.json

```diff
@@ -2,14 +2,14 @@
   "name": "example-dask",
   "image_uri": "public.ecr.aws/saturncloud/saturn:2022.01.06",
   "description": "Samples of code and explanations for common Dask user tasks",
-  "environment_variables": {
-    "SATURN__JUPYTER_SETUP_DASK_WORKSPACE": "true"
+  "extra_packages": {
+    "pip": "lightgbm"
   },
   "working_directory": "/home/jovyan/examples/examples/dask",
   "git_repositories": [
     {
       "url": "https://github.com/saturncloud/examples",
-      "reference": "release-2022.01.06"
+      "reference": "fix-dask"
```

Comment:
I'll change this back before merging, but this is so that integration-tests pick up the right versions of the notebooks.

Reply:
Ok test passed! Switching this back